iree/turbine/kernel/wave/wave.py

```diff
@@ -260,9 +259,6 @@ def _trace_and_get_kernel_signature(
         # Set indices.
         set_node_indices(graph, self.constraints)
 
-        # Analyze Thread Shapes per Op.
-        determine_thread_shapes(graph)
```

Comment:
can we remove the thread_shape_analysis file as well?

Reply:
sure will remove